website/docs/r/application.html.markdown

```diff
@@ -109,7 +109,9 @@ The following arguments are supported:
 
 * `name` - (Required) The display name for the application.
 
-* `homepage` - (optional) The URL to the application's home page. If no homepage is specified this defaults to `https://{name}`.
+* `homepage` - (optional) The URL to the application's home page. This property is now also known as `signInUrl`.
```

Comment:
i don't see a property known as signin url, so maybe this would be better worder as
```suggestion
* `homepage` - (optional) The URL to the application's home page. This property is also referred to as `signInUrl` in [portal?].
```

Reply:
Ah right, I see the Branding blade does actually still show "Home page URL" even though the underlying property changed. I'll amend this.